### PR TITLE
Drop `darwinia-cli`

### PR DESCRIPTION
**Pull request by AurevoirXavier**
1. Companion for https://github.com/darwinia-network/darwinia-common/pull/745
2. Remove `is_darwinia`. Because darwinia is the default option.

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
@AsceticBear @HackFisher, please review.

File: cli/src/command.rs [hunk 1]
```diff
@@ -94,42 +92,31 @@ impl SubstrateCli for Cli {
...
-			)?),
-			path => Box::new(DarwiniaChainSpec::from_json_file(
-				std::path::PathBuf::from(path),
-			)?),
+			path => {
+				let path = PathBuf::from(path);
+				let chain_spec = Box::new(DarwiniaChainSpec::from_json_file(path.clone())?)
+					as Box<dyn ChainSpec>;
+
+				if self.run.force_crab || chain_spec.is_crab() {
```
Comment (HackFisher) [inline cli/src/command.rs:100]: Do we need `chain_spec.is_crab()` here? Is `crab-dev` spec `is_crab` true?

Comment (AurevoirXavier) [reply cli/src/command.rs:100]: Yes. CrabChainSpec shares with all Crab runtimes.

Comment (HackFisher) [reply cli/src/command.rs:100]: crab-dev should not use json file, right?

Comment (AurevoirXavier) [reply cli/src/command.rs:100]: Nope, maybe you want to make a testnet on different machines that you will need to export the chain state to a json file.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**